bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -184,8 +189,14 @@ spec:
             --k8s-service-cidr "${OVN_SVC_CIDR}" \
             --k8s-apiserver "{{.K8S_APISERVER}}" \
             --ovn-config-namespace ${ovn_config_namespace} \
-            --nb-address "tcp://${db_ip}:{{.OVN_NB_PORT}}" \
-            --sb-address "tcp://${db_ip}:{{.OVN_SB_PORT}}" \
+            --nb-address "ssl://${db_ip}:{{.OVN_NB_PORT}}" \
+            --nb-client-privkey /ovn-cert/tls.key \
+            --nb-client-cert /ovn-cert/tls.crt \
+            --nb-client-cacert /ovn-ca/ca-bundle.crt \
+            --sb-address "ssl://${db_ip}:{{.OVN_SB_PORT}}" \
+            --sb-client-privkey /ovn-cert/tls.key \
+            --sb-client-cert /ovn-cert/tls.crt \
+            --sb-client-cacert /ovn-ca/ca-bundle.crt \
```

Comment:
@squeed  These should be passed in the ovnkube-config configmap

Reply:
That needs to pass CI first.